recipes/librdkafka/all/conandata.yml

```diff
@@ -17,36 +17,44 @@ sources:
   "1.6.1":
     url: "https://github.com/edenhill/librdkafka/archive/v1.6.1.tar.gz"
     sha256: "689028b5fdfdce026f396fc2f3bfe9e38947210d9bd22be28c3159cc8c41b57e"
+  "1.7.0":
+    url: "https://github.com/edenhill/librdkafka/archive/v1.7.0.tar.gz"
+    sha256: "c71b8c5ff419da80c31bb8d3036a408c87ad523e0c7588e7660ee5f3c8973057"
 patches:
   "1.3.0":
-    - patch_file: patches/0001-Remove-cmake-module-path-and-change-library-names.patch
+    - patch_file: patches/0001-Remove-cmake-module-path-and-change-library-names-1-3-0.patch
       base_path: source_subfolder
     - patch_file: patches/0002-Change-library-targets-and-result-variables-1-3-0.patch
       base_path: source_subfolder
   "1.4.0":
-    - patch_file: patches/0001-Remove-cmake-module-path-and-change-library-names.patch
+    - patch_file: patches/0001-Remove-cmake-module-path-and-change-library-names-1-3-0.patch
       base_path: source_subfolder
     - patch_file: patches/0002-Change-library-targets-and-result-variables-1-4-0.patch
       base_path: source_subfolder
     - patch_file: patches/0003-Fix-broken-version-1-4-0-on-windows.patch
       base_path: source_subfolder
   "1.5.2":
-    - patch_file: patches/0004-Change-library-names.patch
+    - patch_file: patches/0001-Change-library-names-1-5-2.patch
       base_path: source_subfolder
     - patch_file: patches/0002-Change-library-targets-and-result-variables-1-5-2.patch
       base_path: source_subfolder
   "1.5.3":
-    - patch_file: patches/0004-Change-library-names.patch
+    - patch_file: patches/0001-Change-library-names-1-5-2.patch
```

Comment:
Using a patch 1.5.2 on the "wrong version" it's why they have been just numbered

Reply:
Hmm, I see.  A few thoughts:

- I agree that a specific version number in a patch name is a bit weird when it's applied to newer version, even though it makes sense if it still applies.
- Specifying a version range in the patch name is... unriendly
- numbering as 0004 the first patch is very weird.
- and of course, having identical patch files for every version is wrong.

What about having symlinks that apply to every version? I.e., for 1.5.3 there would be a symlink -1-5-3 that links to -1-5-2.

Reply:
> numbering as 0004 the first patch feels wrong (IMO, more than applying a patch for an old version to a newer version)

🤔 Looking at the history there were patches removed as Conan gained features. We can reset the index if that's a compromise?

> What about having symlinks that apply to every version?

I am a little worried about symlinks being cross-platform, I am not sure how well handled that use case is... but excellent question!

Reply:
I can do the symlinks and we'll see if the windows builds break...

Reply:
I approved, it's not an objective improvement... we'll see if anyone else is as picky as me today =)